src/optim_tests/multivariate/quad_transforms.jl

```diff
@@ -126,21 +126,21 @@ end
 examples["Paraboloid Diagonal"] = _paraboloidproblem(100)
 
 function _randommatrix(N::Int, scaling::Bool=true)
-    F = qrfact(randn(N,N))
+    F = qr(randn(N,N))
     if scaling
-        retval = F[:Q]'*spdiagm(float(1:N))*F[:Q]
+        retval = F.Q'*sparse(Diagonal(float(1:N)))*F.Q
     else
-        retval = F[:Q]'*F[:Q]
+        retval = F.Q'*F.Q
     end
     retval
 end
 
 # TODO: From Julia 0.7 onwards, we can use Base.Test.guardsrand() to restore the existing seed
-oldseed = copy(Base.GLOBAL_RNG) # Store current seed
+oldseed = copy(GLOBAL_RNG) # Store current seed
```

Comment:
You can use `Test.guardsrand` here now.
Example: https://github.com/JuliaLang/julia/blob/master/test/error.jl#L9

Reply:
Interesting